README.md

```diff
@@ -1,2 +1,72 @@
+[![Quortex][logo]](https://quortex.io)
+
 # terraform-aws-storage
+
 A terraform module for Quortex infrastructure AWS persistent storage layer.
+
+It provides a set of resources necessary to provision the bucket and access key on Amazon AWS.
+
+![infra_diagram]
+
+This module is available on [Terraform Registry][registry_tf_aws-eks_cluster].
+
+Get all our terraform modules on [Terraform Registry][registry_tf_modules] or on [Github][github_tf_modules] !
+
+## Created resources
+
+This module creates the following resources in AWS:
+- as many buckets in Amazon S3 as defined in the configuration
+- a new user, with access to the bucket
+- the key ID and secret for this user
+
+
+## Usage example
+
+```
+module "storage" {
+  source = "git::https://github.com/quortex/terraform-aws-storage.git"
+  
+  region         = "eu-west-3"
+
+  buckets        =  ["storage1", "storage2"]
+  storage_prefix = "prf"
+  force_destroy  = true
+  tags           = {}
+}
+
+```
+
+---
+
+## Related Projects
+
+This project is part of our terraform modules to provision a Quortex infrastructure for AWS.
+
+Check out these related projects.
+
+- [terraform-aws-network][registry_tf_aws-eks_network] - A terraform module for Quortex infrastructure network layer.
+
+- [terraform-aws-eks-cluster][registry_tf_aws-eks_cluster] - A terraform module for Quortex infrastructure AWS cluster layer.
+
+- [terraform-aws-eks-load-balancer][registry_tf_aws-eks_load_balancer] - A terraform module for Quortex infrastructure AWS load balancing layer.
+
+- [terraform-aws-storage][registry_tf_aws-eks_storage] - A terraform module for Quortex infrastructure AWS persistent storage layer.
```

Comment:
Not essential, this is the current module

Reply:
Changed as suggested